manifest.json

```diff
@@ -17,19 +17,25 @@
   "name": "Web Thing",
   "options": {
     "default": {
-      "urls": []
+      "urls": [],
+      "poolInterval": 5000
     },
     "schema": {
       "type": "object",
       "required": [
-        "urls"
+        "urls",
+        "poolInterval"
       ],
       "properties": {
         "urls": {
           "type": "array",
           "items": {
             "type": "string"
           }
+        },
+        "poolInterval": {
+          "description": "The interval of updating(pooling) properties from webthing",
```

Comment:
"The interval, in seconds, at which to poll property values."

Reply:
> "The interval, in seconds, at which to poll property values."

In seconds or milliseconds?

Reply:
seconds. milliseconds is too fine-grained for this.